FacebookSwift.xcodeproj/project.pbxproj

```diff
@@ -2340,7 +2353,7 @@
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
 				PRODUCT_BUNDLE_IDENTIFIER = "--PRODUCT-BUNDLE-IDENTIFIER-";
 				SWIFT_OBJC_BRIDGING_HEADER = "$(PROJECT_DIR)/Sources/Core/Internal/FacebookSwift-Bridging-Header.h";
-				SWIFT_VERSION = 4.2;
+				SWIFT_VERSION = 5.0;
```

Comment:
what's this version? is this one sync with objC version ?

Reply:
This was because I missed a checkbox earlier on one of the targets. It's boilerplate. :)